src/dramatiq_abort/middleware.py

```diff
@@ -126,22 +139,32 @@ def after_process_message(
 
     after_skip_message = after_process_message
 
-    def abort(self, message_id: str, abort_ttl: Optional[int] = None) -> None:
+    def abort(
+        self,
+        message_id: str,
+        mode: AbortMode = AbortMode.ABORT,
```

Comment:
Please add `mode` after `abort_ttl` to keep this backward compatible.

Reply:
This wasn't addressed!

it should be:

```python
    def abort(
        self,
        message_id: str,
        abort_ttl: Optional[int] = None,
        mode: AbortMode = AbortMode.ABORT,
    ) -> None
```

Because the keyward arguments are not non-position (`*, abort_ttl...)`, some code might be doing `abort(message_id, 600)`. With your current change, this working code will break.

Reply:
We can also do like you did below and add the non-positional marker (`*`). At least this will allow linter to catch the issue.

Reply:
I just remove the '*' to keep the compatibility and take the correct args order.